README.rst

```diff
@@ -174,3 +181,7 @@ for the complete license.
 .. |coverage| image:: https://codecov.io/github/borgbackup/borg/coverage.svg?branch=master
         :alt: Test Coverage
         :target: https://codecov.io/github/borgbackup/borg?branch=master
+
+.. |screencast| image:: https://asciinema.org/a/28691.png
+        :alt: BorgBackup Installation and Basic Usage
+        :target: https://asciinema.org/a/28691?autoplay=1&speed=2
```

Comment:
I have reverted the order of the sections and added the badge to the other badges. The reference to the asciicast has been placed at the end of the document (which is mostly done for documents) to have the whole info of a reference.